test/integration/assorted.js

```diff
@@ -164,18 +164,30 @@ test('selects .cjs test files', t => {
 	});
 });
 
-test('refuses to load .mjs test files', t => {
+test('refuses to load .mjs test files (node < 13)', t => {
```

Comment:
These tests should be updated to reflect what happens when ESM is not available.

Reply:
You mean two sets of tests one for when ESM is present one without ?

I see the following matrix:

- node 10  - 'AVA cannot yet load ESM files'
- node 12  - 'AVA cannot yet load ESM files'
- node 12 with --experimental-modules - '1 test passed'
- node 13  - '1 test passed'

Or should the tests be conditionally executed based on the presence of the import() function ?

Reply:
You can test for Node versions.

Reply:
So I did,
different outcomes based on the following expression:
```js
Number.parseFloat(process.version.slice(1)) >= 13
```